### PR TITLE
Add wrapInput option.

### PR DESCRIPTION
**Pull request by williamboman**
Make it optional to wrap the input or not. Fixes these kind of issues;

**Before**
Before

**After**
After

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
 <!doctype html>
 <html>
 	<head>
-	
+
```
Comment (williamboman) [inline index.html:4]: My editor did these (remove trailing whitespace) :expressionless:

**Comment by williamboman** [comment]
Ping @erming :)

**Comment by erming** [comment]
Hey, sorry for the delay!
What causes the issue for you? Is it the `position: relative;` that screws your example over?

I'll go ahead and merge.

**Comment by williamboman** [comment]
Well basically the whole wrapping of the `input` with a `div` element caused it.

[I wrote a quick, very ugly, demo.](http://codepen.io/anon/pen/afvEy)

**Comment by williamboman** [comment]
By the way, could you tag a new release on the latest ref, would like to use this in production :)

**Comment by erming** [comment]
Will do!

**Comment by erming** [comment]
Ah, now I see what caused the problem. You're expecting it to work like an inline element. It seems using `display: inline-block;` on the wrapper solves this. I'll experiment with that later (might make it the default behavior).

**Comment by williamboman** [comment]
Yeah. Now when I think about it my fix is actually bad. It should definitely wrap the input, but the wrap element should use `inline-block` in order to imitate the default positioning of an `input` element.

Perhaps dynamically set the `display` CSS property of the wrap element by looking at the `input`?

**Comment by erming** [comment]
If I remember correctly, I had to use a wrapper to position the clone exactly behind the input.

I'm all for getting it to work without the need of any settings. I think that sounds like a good idea!

**Comment by williamboman** [comment]
Yeah, I didn't really think it through. [I reverted my changes and added this.](https://github.com/williamboman/tabcomplete/blob/gh-pages/src/tabcomplete.js#L217)

**Comment by erming** [comment]
Hmm.. That works on the tabcomplete demo `index.html`, but I think input.css("block"); will default to undefined sometimes

**Comment by erming** [comment]
No, not undefined. But this was my result:

Defaults:
- Chrome, Internet Explorer: `inline-block`
- Firefox: `inline`

Try this one: http://jsbin.com/pujodaloho/1/edit?html,js,output

**Comment by williamboman** [comment]
Do you mean `input.css('display')`? Afaik it will always resolve to the browser defaults (`inline-block` on virtually every browser) if there are no rulesets for the element. Browsers with no default properties will default to `undefined`, I suppose.

**Comment by williamboman** [comment]
In the end it will still imitate the `display` property properly, which is the wanted outcome. I think jQuery doesn't set the property if the value is `undefined`.

**Comment by erming** [comment]
I just released `1.5.0`

Time to slack in front of the TV.
Have a good one! :)

**Comment by williamboman** [comment]
Nice, thanks!

**Pull request merged**